PiwikTracker/Tracker.swift

```diff
@@ -63,4 +65,60 @@ final public class Tracker: NSObject {
         }
         self.dispatchTimer = Timer.scheduledTimer(timeInterval: dispatchInterval, target: self, selector: #selector(dispatch), userInfo: nil, repeats: false)
     }
+    
+    var visitor = Visitor.current()
+    var session = Session.current()
+}
+
+// shared instance
+extension Tracker {
+    public static var sharedInstance: Tracker? {
```

Comment:
Great comment?

- [ ] change the shared instance property to `shared` instead of `sharedInstance`